src/Components/Server/src/Circuits/RevalidatingServerAuthenticationStateProvider.cs

```diff
@@ -61,7 +61,7 @@ private async Task RevalidationLoop(Task<AuthenticationState> authenticationStat
         try
         {
             var authenticationState = await authenticationStateTask;
-            if (authenticationState.User.Identity.IsAuthenticated)
+            if (authenticationState.User.Identity?.IsAuthenticated ?? false)
```

Comment:
I think you want `== true` here rather than `?? false`, otherwise the null path gets the same treatment as an authenticated user?

Reply:
The idea here is if authenticationState.User.Identity is not null and IsAuthenticated is true, the condition evaluates to true. Otherwise it evaluates to false.

So if authenticationState.User.Identity is null, the condition will (should?!) evaluate to false.

Reply:
Ah yes - I've just misread the code based on the fact I typically solve this by using `== true` to deal with the type becoming `bool?` rather than defaulting to false (so then the condition fails).

I find the former is more affirmative (_Is this true?_) and clear as otherwise it can be misunderstood at a glance as _"if false"_ (at least, misunderstood by me 😄).

Reply:
No worries. I have no deep feelings for either of the solutions so I am happy to change it.